packages/pocl-job/src/__mocks__/test-2-records.xml

```diff
@@ -0,0 +1,122 @@
+<NewLicence xmlns="EAF1NewLicence.xsd" >
```

Comment:
Did you write a test file generator or can we reuse the one that is already in the jenkins job? What will happen when the GDPR change to the POCL file comes into being - will the loader just ignore it? Should we add it in now?

Reply:
I have not currently implemented a test file generator - these are static and are used in the unit tests.

I was in two minds whether to implement against the current schema or for the upcoming changes but we have not yet received confirmation if/when the schema changes will be made so will address that separately.